File: backend/Vocal.Model/Request/MessageRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Vocal.Model.Request
{
    public class MessageRequest : Request
    {
        public string MessageId { get; set; }
        public string TalkId { get; set; }
    }
}

File: old/src/models/request/messageRequest.ts
import {Request} from './request';

export class MessageRequest extends Request {
    TalkId: string;
    MessageId: string;
    Lang:string;

    constructor(talkId: string, messageId : string, lang: string) {
        super();
        this.TalkId = talkId;
        this.MessageId = messageId;
        this.Lang = lang;
    }
}